apidoc/Titanium/Network/Network.yml

```diff
@@ -568,15 +568,15 @@ properties:
     permission: read-only
     since: {android: "3.6.0", iphone: "1.8.0.1", ipad: "1.8.0.1"}
     platforms: [android, iphone, ipad, macos]
-    osver: {android: {min: "4.1"}, ios: {min: "5.0"}}
+    osver: {android: {min: "4.1"}}
```

Comment:
think we can do the same for the Android part since min is Android 6 (23). But that can be a different PR but I like the idea to remove such old details :+1:

Reply:
Isn't it Android 5 so far?

Reply:
my bad. Was looking at a different package.json :wink: But searching for: `android: {min: "2`, `android: {min: "3`, `android: {min: "4`, `android: {min: "5.0` reveals some stuff that can be removed! Will do that after this one to reduce merge conflicts

Reply:
btw: https://github.com/kaushikgopal/why_bump_android_minsdk to see what features you'll gain (or are always available) when you raise the minSDK.

Reply:
Interesting! Yeah I thought about bumping to 6 as well, but for now the market share of 5 is still to relevant :(